doc/tutorial/customisation.rst

```diff
@@ -0,0 +1,68 @@
+Basic customisation
+====================
+
+HTML
+----
+
+The next step is to manage *how* Sphinx builds the project. It's useful to use
+an HTML theme whose output corresponds better to PDF rendering, so switch to
+Furo; in ``conf.py``, edit the ``html_theme`` setting, to::
+
+    html_theme = 'furo'
+
+Rebuild the HTML to see the effect.
+
+
+PDF
+---
+
+The PDF customisation is a little more complex. We need to do three things.
+
+First, apply some rinoh configuration in ``conf.py``::
+
+    rinoh_documents = [
+        {
+            "doc": "index",
+            "target": "allaboutme",
+            "template": "my-article.rtt"
+        }
+    ]
+
+In brief, this means:
+
+* build a document using the ``index`` file as the root
+* name the output ``allaboutme``
+* use the template configuration file in ``my-article.rtt``.
```

Comment:
Link to the relevant section in the reference using `` :confval:`rinoh_documents` ``